package.json

```diff
@@ -42,6 +45,7 @@
     "uninstall:ios": "echo 'Script not yet added.'",
     "update-env": "curl -H 'Authorization: token '$GITHUB_TOKEN -H 'Accept: application/vnd.github.v3.raw' -O  -L https://api.github.com/repos/rainbow-me/rainbow-env/contents/dotenv && rm -f .env && mv dotenv .env",
     "android-reverse-ports": "adb reverse tcp:8097 tcp:8097 && adb reverse tcp:8081 tcp:8081",
+    "android-reverse-ports-all": "yarn adb-all reverse tcp:8097 tcp:8097 && yarn adb-all reverse tcp:8081 tcp:8081",
```

Comment:
8081 is the default, right? What is 8097, default second device or is it custom or depends on something?

And it only reverses two connections so I guess it only works for 2 devices?

Reply:
no, we need the second post for something else. I think we use this for running the backend locally. 

It doesn't work only for two devices. See `yarn adb-all`